### ISSUE TITLE
Accessibility: Share button not focusable, not useable with keyboard 

### ISSUE THREAD

**Issue by tigexplorer** [LinkStackOrg/LinkStack#410]
Platform: `Ubuntu 22.04 LTS`
Server type: `NGINX/Apache2 ...`
PHP version: `8.1`

[//]: # (Please make sure you have read the system requirements at https://llc.ovh/req before submitting your issue.)
The **Share** Button is not accessible with Tab/ Keyboard. 

So blind users may not able to use it.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#interactive_elements_must_be_focusable
